velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -1065,6 +1065,7 @@ class ParquetRowReader::Impl {
 
   void updateRuntimeStats(dwio::common::RuntimeStatistics& stats) const {
     stats.skippedStrides += rowGroups_.size() - rowGroupIds_.size();
```

Comment:
Not relevant to this PR. But skippedStrides is misleading here since it counts strides not part of the split.
We should ideally add a counter for skipStrides and update it if `rowGroupInRange && (isExcluded || isEmpty))`

Reply:
Thanks for the pointer!